Description: fix build on arm, avoid linking with ldouble FFTW
Author: Matthias Klose <doko@ubuntu.com>
Bug-Debian: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=779724
Last-Update: 2015-03-04
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/setup.py
+++ b/setup.py
@@ -64,8 +64,10 @@
     package_data['pyfftw'] = [
             'libfftw3-3.dll', 'libfftw3l-3.dll', 'libfftw3f-3.dll']
 else:
-    libraries = ['fftw3', 'fftw3f', 'fftw3l', 'fftw3_threads', 
-            'fftw3f_threads', 'fftw3l_threads']
+    libraries = ['fftw3', 'fftw3f', 'fftw3_threads',
+            'fftw3f_threads']
+    if not get_platform().startswith('linux-arm'):
+        libraries.extend(['fftw3l', 'fftw3l_threads'])
 
 class custom_build_ext(build_ext):
     def finalize_options(self):
